autoload/UltiSnips/bootstrap.vim

```diff
@@ -20,10 +22,18 @@ function! UltiSnips#bootstrap#Bootstrap()
        endif
        let g:UltiSnipsUsePythonVersion = "<tab>"
    else
-       if g:UltiSnipsUsePythonVersion == 2
-           let g:_uspy=":py "
-       else
-           let g:_uspy=":py3 "
+       " Use user-provided value, but check if it's available.
```

Comment:
Sorry, my comment was not clear: I meant to expand on the comment in the code so that the next reader has the context we already have from the pull request.